support-frontend/assets/components/checkoutNudge/checkoutNudgeContainer.tsx

```diff
@@ -28,16 +28,47 @@ export function CheckoutNudgeContainer({
 	const [displayNudge, setDisplayNudge] = useState(true);
 
 	const recurringType =
-		abParticipations.singleToRecurringV2 === 'control' ||
-		!abParticipations.singleToRecurringV2
+		abParticipations.singleToRecurringV3 === 'control' ||
+		!abParticipations.singleToRecurringV3
 			? 'MONTHLY'
 			: 'ANNUAL';
 	const currencyGlyph = glyph(detect(countryGroupId));
 	const minAmount = config[countryGroupId][recurringType].min;
-	const paragraphCopyVariant = `Regular, reliable support powers Guardian journalism in perpetuity. If you can, please consider setting up ${
-		recurringType === 'MONTHLY' ? 'a' : 'an'
-	} ${recurringType.toLowerCase()} payment today from just  ${currencyGlyph}${minAmount} – it takes less than a minute.`;
 
+	const minWeeklyAmount =
+		countryGroupId === 'GBPCountries'
+			? Math.ceil((minAmount * 100) / 52).toString() + `p`
+			: currencyGlyph +
+			  (Math.ceil((minAmount * 100) / 52) / 100).toFixed(2).toString();
+
+	const [titleNudge, title2Nudge, paragraphNudge] = copyNudge(
+		recurringType,
+		currencyGlyph + minAmount.toString(),
+		minWeeklyAmount,
+	);
+
+	function copyNudge(
+		recurringType: string,
+		minAmount: string,
+		minWeeklyAmount: string,
+	): string[] {
+		switch (recurringType) {
+			case 'ANNUAL': {
+				return [
+					`Support us every year`,
+					`from just ${minAmount} (${minWeeklyAmount} a week)`,
+					`Funding Guardian journalism every year doesn’t need to be expensive. Make a bigger impact today, and protect our independence long term. Please consider annual support.`,
+				];
+			}
```

Comment:
It would be a good idea to explicitly add a case for MONTHLY to be explicit on what is returned for the MONTHLY case. This function should never hit the default case but the default is needed regardless for the switch.

Reply:
Yep, have added monthly and default now